models/models.js

```diff
@@ -23,6 +24,12 @@ exports.selectUsers = () => {
   });
 };
 
+exports.selectArticles = () => {
```

Comment:
Please break your models and controllers out into individual files within the relevant folder. Eg. Your models folder should have articles.models, topics.models etc and the same for controllers.